docker/build-tools/Dockerfile

```diff
@@ -245,6 +246,15 @@ RUN gem install --no-wrappers --no-document html-proofer -v ${HTMLPROOFER_VERSIO
 RUN gem install --no-wrappers --no-document awesome_bot -v ${AWESOMEBOT_VERSION}
 RUN gem install --no-wrappers --no-document licensee -v ${LICENSEE_VERSION}
 
+# Install su-exec within the ruby-tools context as it contains a build-essentials package group
```

Comment:
Wouldn't it be preferable to put this with the other binary tools and just add the required packages there?

Reply:
I can take a look.